torch/fx/graph_module.py

```diff
@@ -156,8 +156,13 @@ def __init__(self, root: Union[torch.nn.Module, Dict[str, Any]], graph: Graph):
 
             graph (Graph): ``graph`` contains the nodes this GraphModule should use for code generation
 
+            name (str): ``str`` denotes the name of this GraphModule for debugging purposes. If it's unset, all
+                error messages will report as originating from ``GraphModule``; it may be helpful to set this to
+                ``root``'s original name.
```

Comment:
"it may be helpful to set this to ``root``'s original name or a name that makes sense within the context of your transform"